.github/workflows/build-perfetto-ui.yml

```diff
@@ -0,0 +1,30 @@
+name: Build Perfetto UI
+
+permissions:
+  contents: write
+
+on: workflow_dispatch
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set Git Short SHA
+        id: sha
+        run: echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+      - name: Install Dependencies
+        run: tools/install-build-deps --ui
+      - name: Run build
+        run: ui/build
+      - name: Create archive
+        run: zip -r perfetto-ui-${{ steps.sha.outputs.sha_short }}.zip ui/wasm ui/css ui/out/tsc -x "ui/out/tsc/node_modules/*"
+      - name: Upload Perfetto UI
+        if: runner.os == 'Linux'
```

Comment:
I don't think we need this if, because we have no build matrix with other OSes.

Reply:
Good point, I removed it!